46.md

```diff
@@ -127,9 +127,9 @@ Each of the following are methods that the client sends to the remote signer.
 | `get_public_key`         | `[]`                                              | `<hex-pubkey>`                                                         |
 | `nip04_encrypt`          | `[<third_party_pubkey>, <plaintext_to_encrypt>]`  | `<nip04_ciphertext>`                                                   |
 | `nip04_decrypt`          | `[<third_party_pubkey>, <nip04_ciphertext_to_decrypt>]` | `<plaintext>`                                                    |
-| `nip44_get_conversation_key` | `[<third_party_pubkey>]`                       | `<nip44_conversation_key>`                                            |
 | `nip44_encrypt`          | `[<third_party_pubkey>, <plaintext_to_encrypt>]`  | `<nip44_ciphertext>`                                                   |
 | `nip44_decrypt`          | `[<third_party_pubkey>, <nip44_ciphertext_to_decrypt>]` | `<plaintext>`                                                    |
+| `nip44_get_conversation_key`| _Potential future addition_                    |                                                                        |
```

Comment:
Just remove it entirely. There's no point putting future plans in a nip. Either it's defined or it isn't.